code/game/gamemodes/changeling/powers/mutations.dm

```diff
@@ -294,7 +294,7 @@
 	desc = "We turn our skin into tough chitin to protect us from damage."
 	helptext = "Upkeep of the armor requires a low expenditure of chemicals. The armor is strong against brute force, but does not provide much protection from lasers. Retreating the armor damages our genomes. Cannot be used in lesser form."

```

Comment:
Do not forget to change the description to reflect that it is now useful against lasers.